ppr-ui/src/views/mhrInformation/MhrInformation.vue

```diff
@@ -407,12 +410,8 @@ export default defineComponent({
         } else console.log(mhrTransferFiling?.error) // Handle Schema or Api errors here.
       }
 
-      const transferDetailsComponent = context.refs.transferDetailsComponent as any
-
-      // Check if TransferDetails exists on the screen
-      if (transferDetailsComponent) {
-        localState.isTransferDetailsFormValid = await transferDetailsComponent.validateDetailsForm()
-      }
+      // @ts-ignore - function exists
+      await context.refs.transferDetailsComponent.validateDetailsForm()
```

Comment:
Definitely! 
Since then we've tied the Fee Summary / Navigation tool to only render when there is changes made, so the navigation tool that prompts the validation will only exist if the component does.